website_snippet_carousel_product/static/src/scss/s_carousel_product.scss

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Tecnativa - Alexandre Díaz
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+.s_product_carousel {
+    .oe_product {
+        border-width: 0;
+
+        .oe_product_image {
+            position: unset;
+            padding: 15px;
```

Comment:
Can't you use normal bootstrap classes instead? things like a `.col-*` inside a `.row`, etc.

Reply:
This part is a workarround to show the tiles... i don't like the idea of change the 'website_sale.products_item' template.

Reply:
OK, but then try using rem units, as it's the standard in BS4.